.gitpod.yml

```diff
@@ -0,0 +1,42 @@
+# Commands to start on workspace startup
+
+ports:
+  # when port 8000 becomes available open a browser preview
+  # of the running django pattern library instance
+  - port: 8000
+    onOpen: open-preview
+
+tasks:
+  # initialise and install the python part of the app by installing poetry
+  # and the python dependencies
+  - init: |
+      clear
```

Comment:
I don’t get why this is here. Isn’t it valuable to keep the logs from Gitpod?

Reply:
The `clear` command just clears the initial console logged commands from .gitpod.yml. 

I usually add this as I like to see a clean console at the start. I have removed it though because thats just my preference and you are right, someone else might like to see it.